components/Me/index.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import test from 'tape'
+import { configure, shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import { Me } from '../Me'
+
+configure({ adapter: new Adapter() })
+
+test('utils.createActionButton', assert => {
+  assert.plan(1)
+
+  // TODO: Properly hook up withMe/withT here.
```

Comment:
IMO no need to test `withMe/withT` right now

Reply:
maybe a better test case would be popover, clicking close and testing that it is hidden